tools/memap.py

```diff
@@ -98,44 +145,44 @@ def check_new_section_gcc(self, line):
         else:
             return False         # everything else, means no change in section
 
-    
-    def path_object_to_module_name(self, txt):
-        """ Parse a path to object file to extract it's module and object data
+
+    def parse_object_name_gcc(self, line):
```

Comment:
These method names give me the impression that this class should be structured like `mbedToolchain`: A base class that implements the general structure of things, and a subclass for each compiler that implements the compiler specific bits.

Not a blocking comment.